soh/soh/Enhancements/randomizer/3drando/location_access/locacc_fire_temple.cpp

```diff
@@ -203,7 +204,7 @@ void AreaTable_Init_FireTemple() {
 
   areaTable[FIRE_TEMPLE_SCARECROW_ROOM] = Area("Fire Temple Scarecrow Room", "Fire Temple", FIRE_TEMPLE, NO_DAY_NIGHT_CYCLE, {}, {
                   //Locations
-                  LocationAccess(FIRE_TEMPLE_GS_SCARECROW_CLIMB, {[]{return IsAdult || CanChildAttack;}}),
+                  LocationAccess(FIRE_TEMPLE_GS_SCARECROW_CLIMB, {[]{return CanAdultAttack || CanChildAttack;}}),
```

Comment:
```suggestion
                  LocationAccess(FIRE_TEMPLE_GS_SCARECROW_CLIMB, {[]{return CanJumpslash || CanUse(SLINGSHOT) || CanUse(BOOMERANG) || HasExplosives || CanUse(BOW) || CanUse(HOOKSHOT) || CanUse(DINS_FIRE);}}),
```
Another scenario where you cannot get the skulltula with the hammer I believe